pyvista/plotting/plotter.py

```diff
@@ -6885,6 +6885,24 @@ def add_cursor(
 
         return actor
 
+    @property
+    def meshes(self):
+        """Return plotter meshes.
+
+        _extended_summary_
```

Comment:
Could you please explain as I am not understanding the intent of the `_extended_summary_` symbol?

Reply:
That shoudn't be there.